src/frontend/src/flows/manage/index.ts

```diff
@@ -446,17 +452,29 @@ export const devicesFromDeviceDatas = ({
         return acc;
       }
 
-      acc.authenticators.push({
+      const authenticator = {
         alias: device.alias,
         warn: domainWarning(device),
         rename: () => renameDevice({ connection, device, reload }),
         remove: hasSingleDevice
           ? undefined
           : () => deleteDevice({ connection, device, reload }),
-      });
+      };
+
+      if ("browser_storage_key" in device.key_type) {
```

Comment:
I am not a great fan of these kind of imperative code, it's prone to issues because we push within an existing object.

Is there a way to instantiate the object with the appropriate values? e.g. using deconstruction and inner condition or else

Reply:
Are you talking about the usage of `reduce` and pushing into the accumulator? If so, I'd rather not change that (as it was like that before).

Reply:
Ok